cdc/capture/http_handler.go

```diff
@@ -187,7 +190,42 @@ func (h *HTTPHandler) GetChangefeed(c *gin.Context) {
 // @Failure 500,400 {object} model.HTTPError
 // @Router	/api/v1/changefeeds [post]
 func (h *HTTPHandler) CreateChangefeed(c *gin.Context) {
-	// TODO
+	if !h.capture.IsOwner() {
+		h.forwardToOwner(c)
+		return
+	}
+
+	var changefeedConfig model.ChangefeedConfig
+	if err := c.BindJSON(&changefeedConfig); err != nil {
+		c.IndentedJSON(http.StatusInternalServerError, model.NewHTTPError(err))
+		return
+	}
+	log.Info("changefeedConfig", zap.Reflect("config", changefeedConfig))
```

Comment:
Is this used to log request, seems other requests are not logged.

Reply:
Thank you. It should be deleted.